cfme/base/__init__.py

```diff
@@ -55,17 +62,22 @@ def tree_path(self):
         Returns:
             list of path elements for tree navigation
         """
-        current = '' if self.is_slave else ' (current)'
-        # replace two spaces with one, formatting tweak when name is empty
         name_string = f' {self.name} '.replace('  ', ' ')
         path = [
             self.zone.region.settings_string,
             "Zones",
-            f"Zone: {self.rest_api_entity.zone.description} (current)",
-            f"Server:{name_string}[{self.sid}]{current}"  # variables have needed spaces
+            f"Zone: {self.zone.description}{self.zone.current_string}",
```

Comment:
Why change this back to a UI lookup? Its a lot faster with the rest_api_entity use.

Reply:
The Server.zone property uses the rest api already:
```
    @property
    def zone(self):
        entity = self.rest_api_entity
        entity.reload(attributes=['zone'])
        return self.appliance.collections.zones.instantiate(
            name=entity.zone['name'],
            description=entity.zone['description'],
            id=entity.zone['id']
        )
```
If I'm missing something, let me know.